ThirdParty/Bolts/BoltsInClasspath/BoltsInClasspath/ViewController.m

```diff
@@ -0,0 +1,275 @@
+//
+//  ViewController.m
+//  WinObjcBoltsTest
+//
+//  Created by siva kongara on 1/12/17.
+//  Copyright © 2017 venkat kongaraSiva. All rights reserved.
+//
+
+#import "ViewController.h"
+#import "TaskManager.h"
+#import "TaskExecutionResource.h"
+
+
+@interface ViewController () {
+    BFCancellationTokenSource* cancelSource;
+    BFCancellationTokenRegistration* cancelRegistration;
+}
+@end
+
+@implementation ViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view, typically from a nib.
+    [self constructView];
+}
+
+-(void)constructView{
+    
+    [[self view] setBackgroundColor:[UIColor colorWithWhite:0.5 alpha:1.0]];
+    
+    UILabel* label1 = [[UILabel alloc] init];
+    [label1 setText:@"Bolts Sample Application.\nPlease observe console for output."];
```

Comment:
This works well for desktop, but not for mobile device. Consider an overall 'pass' or 'fail' next to the buttons.

I would like to recommend one more trick. Run each positive test when the app starts up as part of constructView. This allows a developer to pick it up, build it, run it, and now that everything is working.